packages/nodes/src/InstructionStatusNode.ts

```diff
@@ -0,0 +1,14 @@
+import type { InstructionStatusNode, InstructionStatus } from '@codama/node-types';
+
+export function instructionStatusNode(
+    status: InstructionStatus,
+    options: { message?: string } = {},
+): InstructionStatusNode {
```

Comment:
I think it'd be nicer to use this helper like this: `instructionStatusNode("deprecated", "Use this other instruction instead")`.

```suggestion
export function instructionStatusNode(
    status: InstructionStatus,
    message?: string,
): InstructionStatusNode {
```

What do you think?

Reply:
yea make sense , thanks